amaranth-chrome-ext/src/CalorieLabeller.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * An object which labels a dish name as low, average, or high-calorie.
+ */
+class CalorieLabeller {
+  /**
+   * Creates a CalorieLabeller object.
+   * @constructor
+   * @param {Map<string, number>} tokenizer A mapping from dish name tokens to
+   * unique integers
+   * @param {tf.LayersModel} model Tensorflow.js layers ML model
+   */
+  constructor(tokenizer, model) {
+    /** @private @const @type {Map<string, number>} */
+    this.tokenizer_ = tokenizer;
+    /** @private @const @type {tf.LayersModel} */
+    this.model_ = model;
+  }
+
+  /**
+   * Labels a single dish as high or low calorie.
+   * @param {string} dishName The name of the dish to label
+   * @return {CalorieLabel} The calorie label for the dish named dishName
+   */
+  label(dishName) {
+    // Step 1: remove special characters from dish name
```

Comment:
Remove these individual step comments as they describe what the code is doing. Comments should almost always add a "why" or "how" rather than a "what".

Reply:
True, some steps could use some more explanation, and some much less

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/f647285d93ec9d9fbeb789cf6e6e452b880fb0b1